File: dbpro/src/main/java/feat/movie/MovieDTO.java
package feat.movie;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import oracle.connect.ColumnName;

import java.sql.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MovieDTO {
	@ColumnName("MOVIE_CODE")
	private String movieCode;

	@ColumnName("MOVIE_TITLE")
	private String movieTitle;

	@ColumnName("MOVIE_DIRECTOR")
	private String movieDirector;

	@ColumnName("MOVIE_AGE")
	private int movieAge;

	@ColumnName("MOVIE_GENRE")
	private String movieGenre;

	@ColumnName("MOVIE_START")
	private Date movieStart;

	@ColumnName("MOVIE_END")
	private Date movieEnd;
}

File: dbpro/src/main/java/kr/ac/sejong/feat/movie/MovieDTO.kt
package kr.ac.sejong.feat.movie

import kr.ac.sejong.config.ColumnName
import java.sql.Date

data class MovieDTO(
    @ColumnName("MOVIE_CODE")
    val movieCode: String? = null,

    @ColumnName("MOVIE_TITLE")
    val movieTitle: String? = null,

    @ColumnName("MOVIE_DIRECTOR")
    val movieDirector: String? = null,

    @ColumnName("MOVIE_AGE")
    val movieAge: Int = 0,

    @ColumnName("MOVIE_GENRE")
    val movieGenre: String? = null,

    @ColumnName("MOVIE_START")
    val movieStart: Date? = null,

    @ColumnName("MOVIE_END")
    val movieEnd: Date? = null
)